darts/tests/models/forecasting/test_deeptime.py

```diff
@@ -0,0 +1,187 @@
+import shutil
```

Comment:
Thanks for these tests! Could you also add some tests to `test_global_forecasting_models.py` (just add the model class in there along the others).